support-e2e/tests/tieredCheckout.test.ts

```diff
@@ -150,7 +150,10 @@ test.describe('Subscribe (S+) incl PromoCode via the Tiered checkout', () => {
 				'Pay £([0-9]+|(([0-9]+).([0-9]+))) per (year|month)',
 			);
 			await page.getByText(paymentButtonRegex).click();
-			await expect(page).toHaveURL(`/uk/thankyou`, { timeout: 600000 });
+			await expect(page).toHaveURL(
+				`/uk/thankyou?promoCode=PLAYWRIGHT_TEST_SPLUS`,
```

Comment:
can/should we strip the promoCode when checking the URL?

Reply:
We could use something like a `startsWith`, but we do expect it to be there, and this tests captures that.